cypress/e2e/course_board/createEditDeleteColumnInBoard.feature

```diff
@@ -1,33 +1,53 @@
+@api_migrated
 @release
 Feature: Course Board - To create, edit and delete column in the course board
 
   As a teacher I want to create, edit and delete the column in the course board so that I can manage the column in the board.
 
   @stable_test
-  Scenario Outline:  Teacher is able to create, edit and delete a column in the course board
+  Scenario: Teacher creates, edites, deletes a column board in the course, including pre-conditions
 
-    #Teacher creates a new course as a pre-condition ---
-    Given I am logged in as a 'teacher1_nbc' at 'nbc'
+    # pre-condition: teacher and student log in to create their account in a same school
+    Given I am logged in as a '<teacher>' at '<namespace>'
+    Given I am logged in as a '<admin>' at '<namespace>'
```

Comment:
In this feature file we don't need `admin` because we need admin only when we have to add student in the course. Otherwise `teacher` is the only user which can be created and used later in the feature file

Reply:
yes but then we need to make another variation as a precondition. So I just used admin way in all tests.